README.md

```diff
@@ -4,14 +4,13 @@ MUSIC BLOCKS
 Music Blocks is a collection of manipulative tools for exploring
 fundamental musical concepts in an integrative and fun way.
 
-<img
```

Comment:
Sir, for most of the images, I can see it this way

![bug1](https://cloud.githubusercontent.com/assets/15171633/24081955/a1265cbc-0ce3-11e7-8bc9-c67fa737cd7c.PNG)

Thus I was working on changing the tag and displaying images.